src/puppetlabs/puppetdb/query_eng.clj

```diff
@@ -252,11 +250,13 @@
     :else obj))
 
 (defn user-query->engine-query
-  ([version query-map options] (user-query->engine-query version query-map options false))
-  ([version query-map options warn-experimental]
+  ([_version query-map options] (user-query->engine-query _version query-map options false))
```

Comment:
Since we are using `version` in this arity, I think it should be without the underscore